janus_server/src/datastore.rs

```diff
@@ -138,6 +138,15 @@ impl<C: Clock> Datastore<C> {
         tx.tx.commit().await?;
         Ok(rslt)
     }
+
+    /// Write a task into the datastore.
+    pub async fn put_task(&self, task: &Task) -> Result<(), Error> {
```

Comment:
I think this should be `test-util` since it always clones the given `Task` at least once. (non-test code _should_, and I believe _does_, wrap the task in an `Arc` & clone that)